packages/investor/src/Investor.ts

```diff
@@ -0,0 +1,12 @@
+import type { ChainTypes } from '@shapeshiftoss/types'
+
+import { InvestorOpportunity } from './InvestorOpportunity'
+
+export interface Investor<C extends ChainTypes, TxType = unknown, MetaData = unknown> {
```

Comment:
doc comments describing how this interface works across multiple investor opportunities (some examples may be useful)